tests/humidity-test.toit

```diff
@@ -29,12 +34,19 @@ main:
   client.close
   provider1.uninstall
 
+  sensor := HumiditySensor
```

Comment:
I get this feeling that having the provider as a possibly extended class, the sensor class as another point of flexibility and two lambdas to govern opening and closing is a bit rich.

Reply:
The subclassing will be removed.